cmd/rpc.go

```diff
@@ -128,7 +128,7 @@ func getToken(path string) (string, error) {
 		fmt.Printf("error getting the JWT secret: %v", err)
 		return "", err
 	}
-	return buildJWTToken(key.Body, perms.AllPerms)
+	return buildJWTToken(key.Body, perms.AllPerms, time.Minute)
```

Comment:
Fine for rpc, I think?

Reply:
No actually, users usually use rpc cli locally or for debugging, I don't think auth token should expire for those purposes. Let's pls leave default behaviour.

Reply:
needs addressing pls - let's not set expiry here.